### PR TITLE
Components new design

### PR DESCRIPTION
**Pull request by nicosampler**
Components for gnosis/safe-react#1648.
- NEW Dot component
- CHANGE: add a strong prop to titles
- CHANGE: add new button type outlined-bordered.
- CHANGE: add a disabled prop to card.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Review by fernandomg — APPROVED** [review]
It looks good so far.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

File: src/dataDisplay/Card/index.tsx [hunk 1]
```diff
@@ -4,22 +4,38 @@ import { rgba } from 'polished';
...
+  position: relative;
+`;
+
+const Disabled = styled.div`
+  opacity: 0.5;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-color: ${({ theme }) => theme.colors.white};
+  z-index: 999;
```
Comment (fernandomg) [inline src/dataDisplay/Card/index.tsx:21]: what's the `z-index` criteria for the whole library?

why it's `999`?

**Review by fernandomg — APPROVED** [review]
:1st_place_medal: LGTM!

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr77--safereactcomponents.review.gnosisdev.com

**Review by dasanra — APPROVED** [review]

**Pull request merged**